scripts/enable-datastore-viewer.sh

```diff
@@ -96,12 +97,15 @@ server {
     }
 }
 "
-	if [ -e /etc/nginx/sites-enabled/${app_id}.conf ]; then
-		cp /etc/nginx/sites-enabled/${app_id}.conf /tmp
-		echo "$pippo" >> /etc/nginx/sites-enabled/${app_id}.conf
-		echo "Datastore viewer enabled for $app_id at http://$(cat /etc/appscale/my_public_ip):$VIEWER_PORT. Allowed IP: $IP."
+
+if [ -e /etc/nginx/sites-enabled/${APP_ID}.conf ]; then
+		cp /etc/nginx/sites-enabled/${APP_ID}.conf /tmp
```

Comment:
Agreed: we don't touch the application nginx configuration, so this backup is not needed anylonger.